src/CashuWallet.ts

```diff
@@ -309,6 +344,36 @@ class CashuWallet {
 		};
 	}
 
+	/**
+	 * Regenerates
+	 * @param count? set number of blinded messages that should be generated
+	 * @param startIndex optionally set starting point for count (default is 0)
+	 * @returns proofs and newKeys if they have changed
+	 */
+	async restore(
+		count: number,
+		startIndex = 0,
```

Comment:
I understand that `startIndex` is optional here, but from a readability aspect, I would use a signature similar to `slice()`: `restore(start, count)` making neither optional.

Reply:
the idea behind this is that a lot of times you might want to check if you have any ecash created at a certain mint with a seedphrase first: `restore(25)` , and then maybe look for more: restore(25,25).

I agree it reads worse, even though it can be a bit more convenient.
I'm down to change it.

Reply:
Another good alternative is using an options object kind of pattern:

`restore(count: number, { startIndex?: number } = {})`